src/madhuraMJun20/BankingSystem.java

```diff
@@ -0,0 +1,56 @@
+package madhuraMJun20;
+
+public class BankingSystem {
+	
+	int counterD=0;
+	int counterC=0;
+	int counterP=0;
+	static int countD=0;
+	static int countC=0;
+	static int countP=0;
+	
+	void setData(int credit, int debit, int print) {
+		
+	}
+	void debit() {
+		countD++;
+		counterD++;
+	}
+	
+	void credit() {
+		countC++;
+		counterC++;
```

Comment:
You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.

Reply:
Done.